### ISSUE TITLE
torch中的梯度操作，如何在tensor中实现呀？

### ISSUE THREAD

**Issue by shuzizihemu** [PaddlePaddle/Paddle#53147]
### 请提出你的问题 Please ask your question

# 在pytorch中
# grad是一个梯度打开的tensor

grad = - delta.grad.detach()
d = delta
...
delta.data = d
delta.grad.zero_()

# 我在Paddle中
grad = - delta.grad.detach()
d = delta
...
delta = d
delta.clear_grad()

这样对吗？

**Comment by lym0302** [comment]
您好，paddle 中 clear_grad 是表示梯度清零，具体两者是否一样，我理解可以自己验证一下
